http/sys_health.go

```diff
@@ -146,10 +146,14 @@ func getSysHealth(core *vault.Core, r *http.Request) (int, *HealthResponse, erro
 		code = sealedCode
 	case replicationState.HasState(consts.ReplicationDRSecondary):
 		code = drSecondaryCode
-	case !perfStandbyOK && perfStandby:
-		code = perfStandbyCode
-	case !standbyOK && standby:
-		code = standbyCode
+	case perfStandby:
+		if !perfStandbyOK {
+			code = perfStandbyCode
+		}
+	case standby:
```

Comment:
Wondering if we want to still allow standbyOK to return 200 on performance standbys? It may be unexpected to stop returning 200 if perf standbys are turned on. What do you think?

Reply:
You're talking about where a customer converts standbys to perf standbys or vice versa?  I mean, I don't entirely understand why we have two different params (standbyOK and perfStandbyOK), but given that we do, if I were setting up monitoring of my vault instances and wanted this feature I'd simply provide both params.

Reply:
@ncabatoff I talked to BK and we're in agreement on this change -- one thing I noticed is that the website docs have some errors around standbyok and perfstandbyok ("load balance" for instance) -- can you give those a once-through as well? After that I think merge this and cherry-pick it back.